src/components/ScrollViewWithPickers.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {ScrollView} from 'react-native';
+
+// eslint-disable-next-line rulesdir/no-inline-named-export
+export const ScrollViewWithPickersContext = React.createContext();
+
+// eslint-disable-next-line react/forbid-foreign-prop-types
+const propTypes = ScrollView.propTypes;
+
+/*
+* <ScrollViewWithPickers /> is a wrapper around <ScrollView /> that provides a ref to the <ScrollView />.
+* <ScrollViewWithPickers /> can be used as a direct replacement for <ScrollView />
+* if it contains one or more <Picker /> / <RNPickerSelect /> components.
+* Using this wrapper will automatically hadnle scrolling to the picker's <TextInput />
+* when the picker modal is opened
+*/
+class ScrollViewWithPickers extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            contentOffsetY: 0,
+        };
+        this.scrollViewRef = React.createRef(null);
+
+        this.handleScroll = this.handleScroll.bind(this);
+    }
+
+    handleScroll(event) {
```

Comment:
Can you please rename this to `setContextScrollPosition`? Just keep in mind we want methods named for what they do, not what they handle. I think this is what the method is doing, feel free to suggest something else that's more accurate.

Reply:
Got it! Will have a closer look at the contributing guidelines, sorry for that!